docs/docsite/rst/user_guide/playbooks_reuse_roles.rst

```diff
@@ -282,6 +282,13 @@ Role dependencies allow you to automatically pull in other roles when using a ro
           dbname: blarg
           other_parameter: 12
 
+Role dependencies may be conditional:
```

Comment:
The section below should be a code snippet - either put a double colon (`::`) at the end of this line, or add a new line with `.. code-block:: yaml` above the `---`. See https://raw.githubusercontent.com/ansible/ansible/devel/docs/docsite/rst/user_guide/intro_inventory.rst for an example.

Reply:
Yeah, sorry, my fingers do Markdown by default and I forgot I was writing rst :).

Reply:
Heh, yeah, that happens to me a lot. +1 for fresh eyes looking at every PR!